README.md

```diff
@@ -0,0 +1,71 @@
+# vui-table
```

Comment:
As the badges as in [vui-typography](https://raw.githubusercontent.com/Brightspace/valence-ui-typography/master/README.md).  So far we've usually been including both NPM and Bower badge which usually seems redundant, but is helpful to spot any instance where they are out of sync (which has happened a few months ago as a result NPM key changes resulting in our publish step failing).  The dependency badge is also quite helpful, and we've become aware of security issues in dependency components.  Either way, we'll want to be consistent with the format of these public components.